package.json

```diff
@@ -98,15 +98,14 @@
   },
   "repository": "https://github.com/bcdevops/nrdk",
   "scripts": {
-    "watcher": "npm i && watch 'npm run build' src",
+    "watcher": "npm ci && watch \"npm run build\" src",
     "prebuild": "oclif-dev manifest",
     "build": "tsc -b",
-    "postpack": "rm -f oclif.manifest.json",
+    "postpack": "rimraf oclif.manifest.json",
     "postbuild": "cpx 'src/**/*.yaml' lib",
```

Comment:
Single quotes needs to be replaced with double quotes. Windows command prompt doesn't understand single quotes.